src/vivarium_gates_mncnh/components/observers.py

```diff
@@ -296,8 +295,8 @@ def configuration_defaults(self) -> dict[str, Any]:
     def __init__(self):
         super().__init__(
             burden_disorders=[
-                f"{NEONATAL_CAUSES.PRETERM_BIRTH}_with_rds",
-                f"{NEONATAL_CAUSES.PRETERM_BIRTH}_without_rds",
+                NEONATAL_CAUSES.PRETERM_BIRTH_WITH_RDS,
```

Comment:
You're creating this list independently here, in the neonatal mortality component, and in the RR observer. I'd suggest storing this list (without "other_causes") somewhere so you only need to update it in one place.